chocolatey/tools/chocolateyInstall.ps1

```diff
@@ -1,49 +1,35 @@
-﻿try {
-    $poshgitPath = join-path (Get-ToolsLocation) 'poshgit'
-
-    try {
-      if (test-path($poshgitPath)) {
-        Write-Host "Attempting to remove existing `'$poshgitPath`'."
-        remove-item $poshgitPath -recurse -force
-      }
-    } catch {
-      Write-Host "Could not remove `'$poshgitPath`'"
-    }
+﻿$ErrorActionPreference = 'Stop'
 
-    $version = "v$Env:chocolateyPackageVersion"
-    if ($version -eq 'v') { $version = 'master' }
-    $poshGitInstall = if ($env:poshGit ) { $env:poshGit } else { "https://github.com/dahlbyk/posh-git/zipball/$version" }
-    $zip = Install-ChocolateyZipPackage 'poshgit' $poshGitInstall $poshgitPath
-    $currentVersionPath = Get-ChildItem "$poshgitPath\*posh-git*\" | Sort-Object -Property LastWriteTime | Select-Object -Last 1
-
-    if(Test-Path $PROFILE) {
-        $oldProfile = @(Get-Content $PROFILE)
-
-        . $currentVersionPath\src\Utils.ps1
-        $oldProfileEncoding = Get-FileEncoding $PROFILE
-
-        $newProfile = @()
-        foreach($line in $oldProfile) {
-            if ($line -like '*PoshGitPrompt*') { continue; }
-
-            if($line -like '. *posh-git*profile.example.ps1*') {
-                $line = ". '$currentVersionPath\profile.example.ps1' choco"
-            }
-            if($line -like 'Import-Module *\src\posh-git.psd1*') {
-                $line = "Import-Module '$currentVersionPath\src\posh-git.psd1'"
-            }
-            $newProfile += $line
-        }
-        Set-Content -path $profile -value $newProfile -Force -Encoding $oldProfileEncoding
-    }
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$moduleName = 'posh-git'  # this may be different from the package name and different case
+
+# module may already be installed outside of Chocolatey
+Remove-Module -Name $moduleName -Force -ErrorAction SilentlyContinue
+
+$sourcePath = Join-Path -Path $toolsDir -ChildPath "$modulename\*"
+$destPath = Join-Path -Path $env:ProgramFiles -ChildPath "WindowsPowerShell\Modules\$moduleName"
```

Comment:
I thought there was Chocolatey support for PowerShell Core?  If so, we need to detect if we are on PS Core and if so, we should probably also install to `"PowerShell\Modules\$moduleName"` if the `$env:ProgramFiles\PowerShell` dir exists.

Reply:
I agree. I have been wondering what to do about whether you run Chocolatey under Core or not and where you should put the modules. However this is purely for the v0 branch so PS Core wasn't a consideration.